PiHole Split-DNS Guide

kqmaverick opened the pull request:
**Description**
Guide for setting up automatic split DNS while still using PiHole.

⚒️ Fixes  #

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

kqmaverick committed:
PiHole Split-DNS Guide

nautilus7 commented:
You need to bump the chart version as well. https://github.com/truecharts/charts/blob/17868d74e64e4330095e508dec23015047c4b8a6/charts/stable/pihole/Chart.yaml#L25

StevenMcElligott commented:
> You need to bump the chart version as well.
> 
> https://github.com/truecharts/charts/blob/17868d74e64e4330095e508dec23015047c4b8a6/charts/stable/pihole/Chart.yaml#L25

Yep anything has to past lint so I can't merge anyways, so basically we bump or @stavros-k approves changes haha

Crow-Control commented:
@kqmaverick Can you add a note highlighting that we advice users to **just** use blocky and that adding pihole decreases the reliability?

kqmaverick committed:
Add unrecommended setup note

Added "The below guide is not the recommened DNS solution for TrueCharts. We recommend running Blocky from the Enterprise train in a standalone configuration. Setting up your DNS as outlined below adds complextiy and will reduce reliability."

Signed-off-by: kqmaverick <121722567+kqmaverick@users.noreply.github.com>

kqmaverick committed:
spelling

Signed-off-by: kqmaverick <121722567+kqmaverick@users.noreply.github.com>

kqmaverick commented:
Added the following to the beginning of the guide.

"The below guide is not the recommended DNS solution for TrueCharts. We recommend running Blocky from the Enterprise train in a standalone configuration. Setting up your DNS as outlined below adds complexity and will reduce reliability."

xstar97 commented on charts/stable/pihole/docs/split-dns.md:
```diff
@@ -0,0 +1,55 @@
+
+:::danger unrecommended configuration
+
+The below guide is not the recommended DNS solution for TrueCharts. We recommend running Blocky from the Enterprise train in a standalone configuration. Setting up your DNS as outlined below adds complexity and will reduce reliability.
```
docusaurus will take the danger part as the 1 liner for doc overview, can you add before that danger tag so it displays a better simpified message?

_Using Blocky in conjunction with PiHole enables seamless automated split DNS functionality._ 

or something along those lines.

Crow-Control commented on charts/stable/pihole/docs/split-dns.md:
```diff
@@ -0,0 +1,55 @@
+# Automatic Split DNS with PiHole
+
```
```suggestion

*Using Blocky in conjunction with PiHole enables seamless automated split DNS functionality.*

```

Commit:
Update charts/stable/pihole/docs/split-dns.md

Signed-off-by: Kjeld Schouten <kjeld@schouten-lebbing.nl>

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
